examples/sol-contract/README.md

```diff
@@ -0,0 +1,21 @@
+# Pyth SDK Example Contract for Solana
+
+This repository contains a simple example demonstrating how to read the Pyth price from the Pyth contract on Solana.
+
+The key functionality of this contract is in the `Loan2Value` function in `src/processor.rs`. 
+This function compares the value of some loan and some collateral, which is important in many lending protocols.
+An example invocation of this contract on the Solana devnet can be found in `scripts/invoke.ts`.
+
+We assume that you have installed `cargo`, `solana`, `solana-keygen`, `npm` and `node`.
+
```

Comment:
Maybe write some more text here explaining what's in the project and where people should look to learn about specific things? E.g., point them to the rust code for the program and where pyth prices are read, maybe mention that we're using the pyth sdk rs, etc.